contributing.md

```diff
@@ -29,3 +12,26 @@ Before edited code can be used on the master branche, a pull request needs to be
 
 #### -issues
 When a (preceived) problem is detected in the code, an issue can be made. This issue will document the problem and make it visible to the team.
+
+# Testing
+## [Mocha:](https://mochajs.org/)
+Mocha is a test executor. This package is in control of all the tests that are to be executed on the code. This framework works with plugins like Sinon, Chai and Sonar that provide the actual tools to test the code, and Mocha makes it possible to use "npm test" in the terminal and collectively execute the tests.
+
+## [Chai:](https://www.chaijs.com/)
+Chai is a framework that makes test conditions easy. Chai makes it possible to make expect, should and assert statements. When one of the expects fails, it can be assumed that there was undefined behaviour.
```

Comment:
"Chai makes it possible to make expect, should and assert statements" -> Chai offers different APIs for BDD-testing. Our convention is to use the expect-variant.